src/features/transactions/components/transactions-table-columns.tsx

```diff
@@ -89,30 +95,63 @@ const fromColumn: ColumnDef<Transaction | InnerTransaction> = {
         : transaction.sender
     return <AccountLink address={from} short={true} />
   },
-  meta: { className: 'max-w-36' },
+  meta: { className: 'max-w-36 align-top' },
 }
 const toColumn: ColumnDef<Transaction | InnerTransaction> = {
   header: transactionToLabel,
   accessorFn: (transaction) => transaction,
-  cell: (c) => <TransactionTo transaction={c.getValue<Transaction>()} />,
-  meta: { className: 'max-w-36' },
+  cell: (c) => {
+    const transaction = c.getValue<Transaction>()
+    if (transaction.type === TransactionType.Payment && transaction.closeRemainder) {
+      return (
+        <div className="grid gap-2">
+          <AccountLink address={transaction.receiver} short={true} />
+          <AccountLink address={transaction.closeRemainder.to} short={true} />
+        </div>
+      )
+    }
+    return (
+      <div className={cn(isAccountCloseTransaction(transaction) ? 'grid' : '')}>
+        <TransactionTo transaction={c.getValue<Transaction>()} />
+      </div>
+    )
+  },
+  meta: { className: 'max-w-36 align-top' },
 }
 const typeColumn: ColumnDef<Transaction | InnerTransaction> = {
   header: 'Type',
   accessorFn: (transaction) => transaction.type,
   cell: (c) => <TransactionTypeBadge transactionType={c.getValue<TransactionType>()} />,
+  meta: { className: 'align-top' },
 }
 const amountColumn: ColumnDef<Transaction | InnerTransaction> = {
   header: transactionAmountLabel,
   accessorFn: (transaction) => transaction,
   cell: (c) => {
     const transaction = c.getValue<Transaction>()
     if (transaction.type === TransactionType.Payment) {
-      return <DisplayAlgo className={cn('justify-center')} amount={transaction.amount} />
+      if (transaction.closeRemainder) {
+        return (
+          <div className="grid gap-2">
```

Comment:
```suggestion
          <div className="grid gap-1">
```
`gap-1` feels spot on to me.